src/csharp/Microsoft.Spark/Utils/CommandSerDe.cs

```diff
@@ -212,7 +225,42 @@ private sealed class UdfWrapperData
 
             foreach (UdfSerDe.FieldData field in fields)
             {
-                SerializeUdfs((Delegate)field.ValueData.Value, curNode, udfWrapperNodes, udfs);
+                SerializeUdfs(
+                    (Delegate)field.ValueData.Value,
+                    curNode,
+                    udfWrapperNodes,
+                    udfs,
+                    udfRefAssemblies);
+            }
+        }
+
+        private static void SerializeReferencedAssemblies(
+            AssemblyName[] referencedAssemblies,
+            Dictionary<string, string> udfRefAssemblies)
+        {
+            foreach (AssemblyName assemblyName in referencedAssemblies)
+            {
+                if (!udfRefAssemblies.ContainsKey(assemblyName.FullName))
+                {
+                    try
+                    {
+                        Assembly asm = Assembly.Load(assemblyName.FullName);
```

Comment:
Should we really be forcing all these assemblies to load?

Reply:
removed method.